### PR TITLE
Add 2.9.0 directory for docs

### PR DESCRIPTION
**Pull request by yves-da**
Copies the current `2.8.0` docs directory to a new `2.9.0` directory in preparation of the upcoming 2.8 release.

### CODE REVIEW

**Review by loiskilberg-da — APPROVED** [review]
Approving with the hopes that in-flight PRs affecting 3.0 can be synced, preferably before this PR is merged.

**Comment by rgugliel-da** [comment]
Can this be merged ?

**Pull request merged**
